lib/trln_argon/controller_override/local_filter.rb

```diff
@@ -33,6 +35,15 @@ def filtered_results_total
           filtered_results_query_response['response']['numFound']
       end
 
+      def filters_applied?
+        search_state.params.key?('f') && !search_state.params.fetch('f').empty? &&
+          filtered_results_total.zero?
```

Comment:
I think `@response.empty?` would be a better way to check this.

Reply:
But with @response passed as an argument to the method.

Reply:
You could also just write the conditional directly in the template since it's not used anywhere else.

Reply:
So in the template you would just have:

```
<% if search_state.params.key?('f') && !search_state.params.fetch('f').empty? && @response.empty? %>